src/frontend/src/pages/FinancePage/FinancePage.tsx

```diff
@@ -62,9 +62,13 @@ const FinancePage = () => {
   const [accountCreditModalShow, setAccountCreditModalShow] = useState<boolean>(false);
   const [showTotalAmountSpent, setShowTotalAmountSpent] = useState(false);
 
+  const [refundSource, setRefundSource] = useState('');
   const [startDate, setStartDate] = useState(new Date());
   const [endDate, setEndDate] = useState(new Date());
```

Comment:
so all of these useStates should be deleted, and moved into actual values in the useForm. The useForm basically acts as a replacement for these, it tracks the values stored in the inputs and allows us to validate certain things about the data. Right now we're storing the data in these useStates and also validating it using the form (not really sure why)